docs/receiving/receiving-with-ingest.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: Using the Ingest API
+---
+
+import CodeTabs from '@theme/CodeTabs';
+import TabItem from '@theme/TabItem';
+
+[Svix Ingest] grants you scalability, security best practices, and
+reliability, all with minimal configuration.
+
+Creating a `Source` will generate an endpoint you can share with a webhook
+provider as a destination for their webhooks.
+
+Ingest supports a variety of signature verification schemes used by webhook
+providers. Supported values for a `Source`'s' `type` include:
+
+- `beehiiv`
+- `brex`
+- `clerk`
+- `github`
+- `guesty`
+- `hubspot`
+- `incidentIo`
+- `lithic`
+- `nash`
+- `pleo`
+- `replicate`
+- `resend`
+- `safebase`
+- `sardine`
+- `shopify`
+- `stripe`
+- `stych`
+- `svix` (that's us!)
+- `zoom`
+
+Additionally there's the option to configure a `Source` as `http` that doesn't
+perform any signature verification at all for cases where the provider has no
+signing scheme, or if _they do_ but it isn't yet supported by Ingest.
+
+:::tip
+Let us know if your webhook provider isn't yet supported so we can add it!
+:::
+
+
+## Create a Source
+
+Getting started with Ingest is a matter of configuring a `Source`.
+
+```sh
+curl https://api.svix.com/ingest/api/v1/source \
+  -H "Authorization: Bearer $SVIX_TOKEN" \
+  -H "Content-Type: application/json" \
+  -d @- << EOF
+{
+  "name": "from-github",
+  "description": "receives webhooks from GitHub",
+  "uid": "from-github",
+  "type": "github",
+  "config": {
+    "secret": "<the signing secret>"
+  }
+}
+EOF
+```
+
+
+## Tell your provider where to send webhooks
+
+The `SourceOut` response will include an `ingestUrl` which is the endpoint you
+give to your provider, telling them where to send their webhooks.
+For GitHub, as is used in this example, this is referred to as the
+_"Payload URL"_.
+
+## Managing incoming messages
+
+Messages received by Ingest are automatically forwarded to a companion Svix
+`Application`.
+
+The `SourceOut` response also includes an `id` field.
```

Comment:
I wonder if we should even mention this yet? We don't expose APIs at the moment
that would allow anyone to query the ingest-app directly.

Reply:
Yeah, I don't know. I wasn't aware of any other way to actually do anything with the messages otherwise. Maybe this is meant to be done in the UI (I have't seen the UI yet).

Reply:
Currently everything is done in the UI, so I would hold off mentioning specifics about the app. Maybe @tasn thinks differently.

Reply:
Okay, I'll take a tour of the UI work and redo this section. I was just going by what I know 🤣

Reply:
I added screenshots of the UI and removed all the mentions of our internal representation.